### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by scriptpfuscher**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/scriptpfuscher/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
```
Comment (scriptpfuscher) [inline ReadMe.md:1]: test

Comment (scriptpfuscher) [inline ReadMe.md:1]: !fix this

Comment (scriptpfuscher) [inline ReadMe.md:1]: !fixed or not

Comment (scriptpfuscher) [inline ReadMe.md:1]: !fix

**Pull request merged**